AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4120,6 +4121,42 @@ function Debug-AzStorageAccountEntraKerbAuth {
                 Write-Error $_
             }
 
+        }
+        #
+        #Check if Fiddler Proxy is cleaned up
+        #
+        if (!$filterIsPresent -or $Filter -match "CheckFiddlerProxy")
+        {   
+           try 
+           {
+                $checksExecuted += 1;
+                
+                $ProxysubFolder = Get-ChildItem -Path Registry::HKLM\SYSTEM\CurrentControlSet\Services\iphlpsvc\Parameters\ProxyMgr
+                foreach($folder in $ProxysubFolder)
+                {
+                    $properties = $folder | Get-ItemProperty
+                    if(($null -ne $properties.StaticProxy) -or ($properties.StaticProxy.Contains("https=127.0.0.1:")))

```

Comment:
I think the condition here should be `-and`, not `-or`. You want to fail if it's set (i.e. not null), and set to `https=127.0.0.1`

```powershell
if (($null -ne $properties.StaticProxy) -and ($properties.StaticProxy.Contains("https=127.0.0.1:")))
```